py/selenium/webdriver/phantomjs/service.py

```diff
@@ -75,13 +75,15 @@ def start(self):
                                             stdout=self._log, stderr=self._log)
 
         except Exception as e:
-            raise WebDriverException("Unable to start phantomjs with ghostdriver.", e)
+            raise WebDriverException(
```

Comment:
The exception will be from `subprocess.Popen`, which is in the same place.
Adding the traceback to the message will it make more noisy (basically duplicating the traceback information).